### ISSUE TITLE
Gets stuck on itself

### ISSUE THREAD

**Issue by TheTechPony** [michaelzangl/minebot#68]
When running two bots with minebuild clear they quite often get stuck on themselves and shake violently.

**Comment by michaelzangl** [comment]
Yes they do.

Try splitting the area so that each of them has it's own working area. The main problem is that they cannot face the block they want to destroy while the other player is in the way.

**Comment by TheTechPony** [comment]
Alright. Do you have any plans to add a failsafe for this in the future?

**Comment by michaelzangl** [comment]
Yes. It will be a bigger update that generally avoids things the bot attempted to do but where it did not succeed. This should also be able to handle invisible blocks (Client thinks it is air, server things the block is still there), both for walking through them and for hitting through.

In this case, the bot would simply find out that it cannot complete the task and walk on to the next position.

For mining, the bot already uses (slight) randomness. I will add that for clear and lumberjack. That way, the two bots will chose different positions to work at after some time.

But it is still best to simply avoid having them interact with each other, the bot cannot communicate with other players.

**Comment by TheTechPony** [comment]
Alright. And something I've noticed with the recent update is that when you have the game minimized the bot won't start mining again. It'll try but it'll time out until you select the window. It didn't do this before.

**Comment by beppe9000** [comment]
The workaround is to bypass crosshair collision with entities.
